extensions/wikia/ArticleVideo/bower_components/html5-skin/js/components/closed-caption/ccPreviewPanel.js

```diff
@@ -0,0 +1,31 @@
+var React = require('react'),
+    Utils = require('../utils'),
+    CONSTANTS = require('../../constants/constants'),
+    ClassNames = require('classnames'),
+    TextTrack = require('../textTrackPanel');
+
+var CCPreviewPanel = React.createClass({
+  render: function(){
+    var closedCaptionPreviewTitle = Utils.getLocalizedString(this.props.language, CONSTANTS.SKIN_TEXT.CLOSED_CAPTION_PREVIEW, this.props.localizableStrings);
+    var closedCaptionSampleText = Utils.getLocalizedString(this.props.closedCaptionOptions.language, CONSTANTS.SKIN_TEXT.SAMPLE_TEXT, this.props.localizableStrings);
+    if (!closedCaptionSampleText) closedCaptionSampleText = Utils.getLocalizedString('en', CONSTANTS.SKIN_TEXT.SAMPLE_TEXT, this.props.localizableStrings);
+
+    var previewCaptionClassName = ClassNames({
+      'oo-preview-caption': true,
+      'oo-disabled': !this.props.closedCaptionOptions.enabled
+    });
+    var previewTextClassName = ClassNames({
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the declaration of the unused 'previewTextClassName' variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedVariable)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "previewTextClassName" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)